client/events.lua

```diff
@@ -7,6 +7,15 @@ RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
         SetCanAttackFriendly(cache.ped, true, false)
         NetworkSetFriendlyFireOption(true)
     end
+
+    local template = GetConvar('qbx:motdTemplate', '')
+    local args = json.decode(GetConvar('qbx:motdArgs', '[]'))
```

Comment:
Is there a way we can make the motd a single string convar? This feels a bit too in the weeds, I assume people could write the style inline if it were a single string?

Reply:
That's true, we can make it the template only without the args.

Reply:
Ah and as for the inline style, that doesn't work inside args. The only way to do a single convar is if we make it template-only.

Reply:
Nice. I think that's a better way to do it.